controller/space.go

```diff
@@ -64,6 +65,9 @@ func NewSpaceController(
 		resourceManager:   resourceManager,
 		DeploymentsClient: http.DefaultClient,
 		CodebaseClient:    http.DefaultClient,
+		ClientGetter: &defaultClientGetter{
```

Comment:
I did it this way because the new CanGet*() API's are internal, and it seemed more efficient to call directly.  However, it's messy in several ways: I had to pass in config.Registry instead of SpaceConfiguration, and it uses the internal CanGetSpace() call, which is ... internal.  So I felt slightly dirty coding it.

If I understand it, your proposal is to call //deployments/Space/{spaceID}?qp=true under the hood and use the output of that?

Reply:
We could add a parameter to only get the root space object, but maybe I should just leave that in a comment?